crypto.go

```diff
@@ -27,8 +27,8 @@ type DHPair interface {
 	PublicKey() Key
 }
 
-// Key is any 32-byte key. It's created for the possibility of pretty hex output.
-type Key [32]byte
+// Key is any 33-byte key. It's created for the possibility of pretty hex output.
```

Comment:
Why do we extended it by 1 byte?

Reply:
sorry, that's a left over, we change it to a slice, this implementation uses 32 bytes keys, while we need 33 bytes